damnit/ctxsupport/ctxrunner.py

```diff
@@ -465,8 +465,7 @@ def save_hdf5(self, hdf5_path, reduced_only=False):
 
                 if (
                     isinstance(data, (np.ndarray, xr.DataArray))
-                    and data.ndim == 1
-                    and data.shape[0] > 1
+                    and data.size > 1
                 ):
```

Comment:
I would guess that arrays with 2 or more dimensions will almost always be changing, so making the summary figure bold may not be as useful. Let's go with it and see what happens - but I think it might support my case for letting the user override whether values are bold, which came up in discussing #221.